<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>VisualDesigner</class>
    <widget class="QWidget" name="VisualDesigner">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>600</width>
                <height>420</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>VisualDesigner</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
                <widget class="QTreeView" name="treeView_Project"/>
            </item>
            <item>
                <widget class="QStackedWidget" name="stackedWidget">
                    <property name="currentIndex">
                        <number>1</number>
                    </property>
                    <widget class="QWidget" name="page">
                        <layout class="QVBoxLayout" name="verticalLayout">
                            <item>
                                <widget class="QGroupBox" name="groupBox">
                                    <property name="title">
                                        <string>枚举</string>
                                    </property>
                                    <layout class="QGridLayout" name="gridLayout">
                                        <item row="0" column="0">
                                            <widget class="QLabel" name="label">
                                                <property name="text">
                                                    <string>名称</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item row="0" column="1">
                                            <widget class="QLineEdit" name="lineEdit"/>
                                        </item>
                                        <item row="1" column="0">
                                            <widget class="QLabel" name="label_2">
                                                <property name="text">
                                                    <string>完整名称</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item row="1" column="1">
                                            <widget class="QLineEdit" name="lineEdit_2">
                                                <property name="text">
                                                    <string/>
                                                </property>
                                            </widget>
                                        </item>
                                        <item row="2" column="0" colspan="2">
                                            <widget class="QTableWidget" name="tableWidget"/>
                                        </item>
                                        <item row="3" column="1">
                                            <spacer name="verticalSpacer">
                                                <property name="orientation">
                                                    <enum>Qt::Vertical</enum>
                                                </property>
                                                <property name="sizeHint" stdset="0">
                                                    <size>
                                                        <width>20</width>
                                                        <height>40</height>
                                                    </size>
                                                </property>
                                            </spacer>
                                        </item>
                                    </layout>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QWidget" name="page_2">
                        <layout class="QVBoxLayout" name="verticalLayout_2">
                            <item>
                                <widget class="QGroupBox" name="groupBox_2">
                                    <property name="title">
                                        <string>结构体</string>
                                    </property>
                                    <layout class="QGridLayout" name="gridLayout_2">
                                        <item row="0" column="0">
                                            <widget class="QLabel" name="label_3">
                                                <property name="text">
                                                    <string>名称</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item row="0" column="1">
                                            <widget class="QLineEdit" name="lineEdit_3"/>
                                        </item>
                                        <item row="1" column="0">
                                            <widget class="QLabel" name="label_4">
                                                <property name="text">
                                                    <string>完整名称</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item row="1" column="1">
                                            <widget class="QLineEdit" name="lineEdit_4">
                                                <property name="text">
                                                    <string/>
                                                </property>
                                            </widget>
                                        </item>
                                        <item row="2" column="0">
                                            <widget class="QLabel" name="label_5">
                                                <property name="text">
                                                    <string>继承</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item row="2" column="1">
                                            <widget class="QLineEdit" name="lineEdit_5">
                                                <property name="text">
                                                    <string/>
                                                </property>
                                            </widget>
                                        </item>
                                        <item row="3" column="0" colspan="2">
                                            <widget class="QTableWidget" name="tableWidget_2"/>
                                        </item>
                                        <item row="4" column="1">
                                            <spacer name="verticalSpacer_2">
                                                <property name="orientation">
                                                    <enum>Qt::Vertical</enum>
                                                </property>
                                                <property name="sizeHint" stdset="0">
                                                    <size>
                                                        <width>20</width>
                                                        <height>38</height>
                                                    </size>
                                                </property>
                                            </spacer>
                                        </item>
                                    </layout>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
